CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ ENHANCEMENTS:
 
 * access_identity_provider: add `claims` and `scopes` fields ([#1237](https://github.com/cloudflare/cloudflare-go/issues/1237))
 
+BUG FIXES:
```

Comment:
this file is automatically generated on merge so this change doesn't need to be here